sheets/3/exercise-2.tex

```diff
@@ -77,31 +69,43 @@
 \end{center}
 Ein paar Beispiele:
 \begin{lstlisting}
-if ( neo.canMove() && (neo.countCoins==7) ){
+if ( neo.canMove() && (neo.getCoinCountOnField()==7) ){

```

Comment:
Hat neo diese Methode? Ansonsten den code (neo.getCurrentlyCollectableCoins().size()) von der vorherigen Aufgabe benutzen!

Genauso in den Beispielen darunter!